src/model/CodacyInput.ts

```diff
@@ -0,0 +1,41 @@
+export type ParameterValue = any
```

Comment:
argh :-(

Reply:
Can be anything representable as Json value.. That's why `any`

Reply:
argh:
https://github.com/microsoft/TypeScript/issues/1897
it's not yet encoded in std lib :-(